S/Spot_julia/build_tarballs.jl

```diff
@@ -19,7 +20,19 @@ delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/spot-2.9.7/
+if [[ ("${target}" == x86_64-apple-darwin*) ]]; then
+    # LLVM 15 requires macOS SDK 10.14, see
+    # <https://github.com/JuliaPackaging/Yggdrasil/pull/5592#issuecomment-1309525112> and
+    # references therein.
+    pushd $WORKSPACE/srcdir/MacOSX10.*.sdk
+    rm -rf /opt/${target}/${target}/sys-root/System
+    cp -ra usr/* "/opt/${target}/${target}/sys-root/usr/."
+    cp -ra System "/opt/${target}/${target}/sys-root/."
+    export MACOSX_DEPLOYMENT_TARGET=10.14
+    popd
+fi
+
+cd $WORKSPACE/srcdir/spot*
 ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-python
 make -j${nproc}
 make install
```

Comment:
```suggestion
make install
install_license COPYING
```

Reply:
I think I had copied the GNU license from the original src in the wrapper I wrote: https://github.com/MaximeBouton/spot_julia/blob/main/LICENSE.md 

Do you know if I should remove that one?